### PR TITLE
Renaming mesh commands params for acheiving parity with az mesh commands

### PR DESCRIPTION
**Pull request by rapatchi**
Fixes # .

Changes include:

-
-
-

Verified:

- [x] Build and test CI passes
- [x] History and readme updated to reflect changes
- [ ] Package version updated according to semantic versioning rules
- [x] Tests modified or added, when applicable
- [x] Updated code owners file, when applicable
- [x] Read the [PR checklist](https://github.com/Azure/service-fabric-cli/wiki/Checklist)

### CODE REVIEW

File: src/sfctl/tests/request_generation_test.py [hunk 1]
```diff
@@ -1083,28 +1076,28 @@ def paths_generation_helper(self):  # pylint: disable=too-many-statements, too-m
...
-            'mesh secretvalue list --secret-resource-name some~secret~resource~name',
+            'mesh secretvalue list --secret-name some~secret~resource~name',
             'GET',
             '/Resources/Secrets/some~secret~resource~name/values',
             ['api-version=6.4-preview']
         )
 
         self.validate_command( # show secretvalue show value
-            'mesh secretvalue show --secret-resource-name some~secret~resource~name --secret-value-resource-name secret~value~name --show-value',
+            'mesh secretvalue show --secret-name some~secret~resource~name --version secret~value~name --show-value',
```
Comment (Christina-Kang) [inline src/sfctl/tests/request_generation_test.py]: nit: secret~value~name -> secret~value~version

File: src/sfctl/tests/request_generation_test.py [hunk 2]
```diff
@@ -1083,28 +1076,28 @@ def paths_generation_helper(self):  # pylint: disable=too-many-statements, too-m
...
-            'mesh secretvalue show --secret-resource-name some~secret~resource~name --secret-value-resource-name secret~value~name',
+            'mesh secretvalue show --secret-name some~secret~resource~name --version secret~value~name',
             'GET',
             '/Resources/Secrets/some~secret~resource~name/values/secret~value~name',
             ['api-version=6.4-preview']
         )
 
         self.validate_command( # delete secretvalue resource
-            'mesh secretvalue delete --secret-resource-name some~secret~resource~name --secret-value-resource-name secret~value~name',
+            'mesh secretvalue delete --secret-name some~secret~resource~name --version secret~value~name',
```
Comment (Christina-Kang) [inline src/sfctl/tests/request_generation_test.py]: nit: secretvaluename -> secretvalueversion

File: src/sfctl/tests/request_generation_test.py [hunk 3]
```diff
@@ -1083,28 +1076,28 @@ def paths_generation_helper(self):  # pylint: disable=too-many-statements, too-m
         )
 
         self.validate_command( # show secretvalue resource
-            'mesh secretvalue show --secret-resource-name some~secret~resource~name --secret-value-resource-name secret~value~name',
+            'mesh secretvalue show --secret-name some~secret~resource~name --version secret~value~name',
```
Comment (Christina-Kang) [inline src/sfctl/tests/request_generation_test.py]: nit: secretvaluename -> secretvalueversion

File: src/sfctl/params.py [hunk 4]
```diff
@@ -209,3 +209,53 @@ def custom_arguments(self, _):  # pylint: disable=too-many-statements
         # expect the parameter command_input in the python method as --command in commandline.
         arg_context.argument('command_input',
                              CLIArgumentType(options_list='--command'))
+
+    with ArgumentsContext(self, 'mesh') as arg_context:
+        arg_context.argument('application_resource_name',
+                             options_list=('--app-name', '--application-name'),
+                             help="The name of the application", id_part='application_name')
```
Comment (Christina-Kang) [inline src/sfctl/params.py]: What does id_part do?

Comment (rapatchi) [reply src/sfctl/params.py]: This shouldn't have been there removing it

File: src/sfctl/tests/request_generation_test.py (hunk 3 above)
Comment (rapatchi) [reply src/sfctl/tests/request_generation_test.py]: renamed

File: src/sfctl/tests/request_generation_test.py (hunk 2 above)
Comment (rapatchi) [reply src/sfctl/tests/request_generation_test.py]: renamed

File: src/sfctl/tests/request_generation_test.py (hunk 1 above)
Comment (rapatchi) [reply src/sfctl/tests/request_generation_test.py]: done

**Pull request merged**